doc/changes/latest.inc

```diff
@@ -175,6 +175,8 @@ API
 
 - The method ``stc_mixed.plot_surface`` for a :class:`mne.MixedSourceEstimate` has been deprecated in favor of :meth:`stc.surface().plot(...) <mne.MixedSourceEstimate.surface>` by `Eric Larson`_
 
-- Add ``use_dev_head_trans`` parameter to :func:`mne.preprocessing.annotate_movement` to allow choosing the device to head transform is used to define the fixed cHPI coordinates By `Luke Bloy`_
+- Add ``use_dev_head_trans`` parameter to :func:`mne.preprocessing.annotate_movement` to allow choosing the device to head transform is used to define the fixed cHPI coordinates by `Luke Bloy`_
 
 - The function ``mne.channels.read_dig_captrack`` will be deprecated in version 0.22 in favor of :func:`mne.channels.read_dig_captrak` to correct the spelling error: "captraCK" -> "captraK", by `Stefan Appelhoff`_
+
+- The default argument ``meg=True`` in func:`mne.io.pick_types` will change to ``meg=False`` in version 0.22 by `Clemens Brunner`_
```

Comment:
This is documented at the root level so this will be an error

https://mne.tools/stable/generated/mne.pick_types.html
```suggestion
- The default argument ``meg=True`` in func:`mne.pick_types` and related methods like :meth:`mne.io.Raw.pick_types` and :meth:`mne.Evoked.pick_types` will change to ``meg=False`` in version 0.22 by `Clemens Brunner`_
```

Reply:
Just so I understand, where will this be an error (if the doc contains func:`mne.io.pick_types`)?

Reply:
> where will this be an error

CircleCI will say something about not being able to link to the ref properly, and the build will fail

Reply:
(alternatively, locally you can do `make html_dev-noplot` in the `doc/` directory then `echo $?` afterward it should not be zero, and if you look up you should see a warning about not being able to link properly)

Reply:
... though I seem to remember your previous build being green, so maybe it figures out to go up to `mne.pick_types` when the `mne.io.pick_types` link is requested somehow?

Reply:
The previous CircleCI build was all green with exit code 0, yes.

Reply:
Looks like it [was indeed green](https://app.circleci.com/pipelines/github/mne-tools/mne-python/1839/workflows/5bbacad8-9713-404d-b997-25aa61ef8818/jobs/20306/steps) but shouldn't have been because the [link did not get created correctly](https://20306-1301584-gh.circle-artifacts.com/0/dev/whats_new.html#api), maybe some latest Sphinx change made this fail to link without raising an error, not sure...

Reply:
the linking failure is due to lack of a colon at the beginning of `func`

Reply:
Grrr... thanks @drammock . I'll see if changing our `default_role` to something other than `"autolink"` linke `"py:obj"` would have made this error show up, in which case we should change it

Reply:
I've inserted the missing colon and now use `mne.pick_types`.